src/tests/access/test_dual_ownable.cairo

```diff
@@ -66,14 +66,15 @@ fn test_dual_owner() {
 }
 
 #[test]
+#[ignore] // TODO: Enable test once it's supported
```

Comment:
We have the discussion opened [here](https://github.com/OpenZeppelin/cairo-contracts/pull/1027#discussion_r1661714103) regarding whether to add these comments. Let's settle before merging.

Reply:
Updated comments to the suggested format